Log Viewing/ARKLogTableViewController.m

```diff
@@ -481,10 +481,6 @@ - (void)_scrollTableViewToBottomAnimated:(BOOL)animated;
 
 @implementation ARKTimestampLogMessage
 
-@synthesize text = _text;
```

Comment:
Actually, there doesn't seem to be a good reason for `ARKTimestampLogMessage` to be declared in a testing header. Might make sense to clean that up while you're here.